test/msw-api/res/network-error.test.ts

```diff
@@ -0,0 +1,16 @@
+import * as path from 'path'
+import { TestAPI, runBrowserWith } from '../../support/runBrowserWith'
+
+let runtime: TestAPI
+
+beforeAll(async () => {
+  runtime = await runBrowserWith(
+    path.resolve(__dirname, 'network-error.mocks.ts'),
+  )
+})
+
+afterAll(() => runtime.cleanup())
+
+test('res.networkError causes Fetch API to throw error', async () => {
+  expect(runtime.request({ url: `${runtime.origin}/user` })).toThrow()
```

Comment:
Does this behavior satisfies your network failure mocking scenario?

Reply:
Currently the tests fail because the `networkError` function returns an empty object instead of throwing. I don't know where the empty object is coming from. Could you point me in the right direction?